Extractor for emission line luminosities from narrow line regions of galaxies

sachiwee opened the pull request:
This nodes.property_extractor class computes narrow line region luminosities of galaxies by interpolating galaxy’s metallicity, and the AGN's ionization parameter, and spectral index from the Cloudy grid assuming constant density 100 cm^-3

abensonca reviewed (approved):
This all looks good. I cleaned up some formatting and variable names. 

Two things for you to check.

1. The variable `halfMassRadius` was computed but not actually used anywhere. So I removed it, but I wanted to check if this is correct - was it supposed to be used somewhere in the calculation?

2. I used the `atomicRecombinationRateRadiative` class to get the hydrogen recombination coefficient. The value should be very close to what you used, but check that the results haven't changed too much.

If that all seems ok I'll merge this once tests have passed.

sachiwee commented:
Hi Andrew,
Thank you so much!
1. Yes, I think you suggested we could use half Mass radius as an approximate to stronmgren radius when we were still in the early stages. We don’t need it anymore. Thanks so much for removing it.
2. Great! I will check this and let you know!
Thanks so much!

> On Mar 21, 2025, at 2:24 PM, Andrew Benson ***@***.***> wrote:
> 
> 
> @abensonca approved this pull request.
> 
> This all looks good. I cleaned up some formatting and variable names.
> 
> Two things for you to check.
> 
> The variable halfMassRadius was computed but not actually used anywhere. So I removed it, but I wanted to check if this is correct - was it supposed to be used somewhere in the calculation?
> 
> I used the atomicRecombinationRateRadiative class to get the hydrogen recombination coefficient. The value should be very close to what you used, but check that the results haven't changed too much.
> 
> If that all seems ok I'll merge this once tests have passed.
> 
> —
> Reply to this email directly, view it on GitHub <https://github.com/galacticusorg/galacticus/pull/818#pullrequestreview-2707290994>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/ALEPJ4LOKSDIE4B3CZSRPCT2VR7RBAVCNFSM6AAAAABZOXKJHOVHI2DSMVQWIX3LMV43YUDVNRWFEZLROVSXG5CSMV3GSZLXHMZDOMBXGI4TAOJZGQ>.
> You are receiving this because you authored the thread.
>
